### PR TITLE
move onward journey for videos to own component

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What does this change?

While trying to figure out what the devil is slowing things down with ads, I keep coming up against not really knowing what's going on in this file.

I am going to try break it into a more coherent structure and flow.

We will need to break this up at some stage as having a new player would not remove the need to have all the furniture, so this isn't wasted effort (I hope 🙏).
## What is the value of this and can you measure success?

I can actually do some work on the player.
## Does this affect other platforms - Amp, Apps, etc?

Nope
## Screenshots

See site, they're the same
## Request for comment

@akash1810

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/video/more-in-series-container.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    'common/modules/component',
+    'common/utils/mediator'
+], function(
+    Component,
+    mediator
+) {
+
+    function init(el, mediaType, section, shortUrl, series) {
+        var component = new Component();
+        var endpoint  = '/' + mediaType + '/section/' + section +
```
Comment (akash1810) [inline static/src/javascripts/projects/common/modules/video/more-in-series-container.js]: :'( string interpolation

**Comment by akash1810** [comment]
👍

File: static/src/javascripts/bootstraps/enhanced/media/main.js [hunk 2]
```diff
@@ -362,43 +364,31 @@ define([
...
-        }
-
-        section.endpoint = endpoint;
-
-        section.fetch(attachTo).then(function () {
-            mediator.emit('page:media:moreinloaded', attachTo);
-            mediator.emit('page:new-content', attachTo);
-        });
+        var el  = $('.js-onward')[0];
+        moreInSeriesContainer.init(
```
Comment (johnduffell) [inline static/src/javascripts/bootstraps/enhanced/media/main.js]: are you sure it's more in series? I had it as more in section (that's specifically *not* in the series)

Comment (johnduffell) [inline static/src/javascripts/bootstraps/enhanced/media/main.js]: the more in series one goes in the normal onward trails.js - https://github.com/guardian/frontend/blob/master/static/src/javascripts/bootstraps/enhanced/trail.js#L94

**Pull request merged**